tree_hugger/core/parser/cpp/cpp_parser.py

```diff
@@ -0,0 +1,169 @@
+import re
+from typing import List, Dict
+from pathlib import Path
+
+from tree_sitter import Tree, Node, TreeCursor
+
+from tree_hugger.core.code_parser import BaseParser, match_from_span
+from tree_hugger.core.queries import Query
+
+
+class CPPParser(BaseParser):
+	
+    QUERY_FILE_PATH = Path(__file__).parent / "queries.yml"
```

Comment:
I was thinking, this line comes back to each `<Language>Parser` class. Maybe we can off-load it to `BaseParser` after all